plugin/src/main/java/net/dzikoysk/funnyguilds/feature/invitation/guild/GuildInvitation.java

```diff
@@ -0,0 +1,25 @@
+package net.dzikoysk.funnyguilds.feature.invitation.guild;
+
+import java.util.UUID;
+import net.dzikoysk.funnyguilds.feature.invitation.Invitation;
+import net.dzikoysk.funnyguilds.guild.Guild;
+import net.dzikoysk.funnyguilds.guild.GuildManager;
+import net.dzikoysk.funnyguilds.user.User;
+import net.dzikoysk.funnyguilds.user.UserManager;
+import panda.std.Option;
+
+public class GuildInvitation extends Invitation<Guild, User> {
```

Comment:
Czy `GuildInvitation` nie może przyjmować po prostu jako parametry `Guild from, User to`?